spec/requests/api/v1/posters_requests_spec.rb

```diff
@@ -165,4 +165,31 @@
     expect(poster).to have_key(:img_url)
     expect(poster[:img_url]).to be_a(String)
   end
+
+  it "can create a poster" do
+    poster_params = {
+      name: "REGRET",
+      description: "Hard work rarely pays off.",
+      price: 89.0,
+      year: 2018,
+      vintage: true,
+      img_url: "https://plus.unsplash.com/premium_photo-1661293818249-fddbddf07a5d"
+    }
+
+    headers = {"CONTENT_TYPE" => "application/json"}
+
+    post "/api/v1/posters", headers: headers, params: JSON.generate(poster: poster_params)
+
+    newposter = Poster.last
```

Comment:
Strictly for consistency, try to make sure your variable names follow a naming convention:
newPoster or new_poster

Otherwise, the rest of your code looks great!

Reply:
Thanks! I couldn't think of anything off the top of my head but we might wanna change that variable to something better if we have time.

Reply:
There's always gonna be refactor opportunities. I'm not too pressed about it at the moment!